instance/migrations/0050_appserver_refactor3.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import django_extensions.db.fields
+import instance.models.instance
+import instance.models.mixins.utilities
+import instance.models.utils
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('instance', '0049_appserver_refactor2'),
+    ]
+
+    operations = [
+        # TODO: Uncomment this once deemed safe
```

Comment:
I was going to leave this commented so that we'll be able to reverse the migrations if necessary
(I was inspired by the edX policy which is that migrations which delete tables must always be deployed in a later release from the data migration; otherwise devops can't easily undo the release if needed)... 

But I decided it will be better to carefully test on stage and to backup using [this SQL](https://gist.github.com/bradenmacdonald/148240764a1aede24ba50fae24119c7b) instead.

So I will fix that. Note: reverse this one migration before checking out my latest commits :)